src/index.js

```diff
@@ -1,47 +1,64 @@
-import Mario from './arts/mario.ansi';
```

Comment:
What do you think if I add one more script in `package.json` for this?
So simply we need to copy a bunch of files to `dist` folder:

```
"scripts": {
    "build": "webpack -p && npm run copy-arts",
    "watch": "webpack --watch --progress && npm run copy-arts",
    "copy-arts": "cp -R src/arts/ dist/arts/"
  }
```

It can be more convenient to do it this way. Or you still wanna add them via `import`?